jboss-logmanager-ecs-formatter/src/test/java/co/elastic/logging/jboss/logmanager/EcsFormatterTest.java

```diff
@@ -0,0 +1,211 @@
+/*-
+ * #%L
+ * Java ECS logging
+ * %%
+ * Copyright (C) 2019 - 2020 Elastic and contributors
+ * %%
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * #L%
+ */
+package co.elastic.logging.jboss.logmanager;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.PrintWriter;
+import java.time.Instant;
+import java.util.Map;
+
+import org.jboss.logmanager.ExtLogRecord;
+import org.jboss.logmanager.Level;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class EcsFormatterTest {
```

Comment:
Isn't most of this already tested by `JBossLogManagerTest`? The exact matching on the JSON structure in this test is a bit brittle. If, for example, a new field is added all tests will fail.

Reply:
I don't know what is JBossLogManagerTest is testing. But you are most probably right that there is an overlap.

Basicly I did the same as for the JUL ECS Formatter. There is also a EcsFormatterTest and JBossLogManagerTest. Do yo think I should remove EcsFormatterTest?

Reply:
If you feel there’s something important that’s not covered in JBossLogManagerTest it might make sense to add it to the abstract test. If you want to test stuff that’s really specific to JBoss log manager just leave those tests and remove the ones that duplicate. When in doubt, lean towards lowering complexity.

Reply:
I removed a couple of tests from EcsFormatterTest which I think are already covered by JBossLogManagerTest.